src/Extensions/ExtHostClient.re

```diff
@@ -85,12 +87,12 @@ let start =
     | ("MainThreadStatusBar", "$setEntry", args) =>
       In.StatusBar.parseSetEntry(args) |> apply(onStatusBarSetEntry);
       Ok(None);
-    | (s, m, _a) =>
-      Log.error(
-        Printf.sprintf(
+    | (system, method, _a) =>
+      Log.errorf(m => 
+        m(
```

Comment:
I have no idea what `s` and `m` is here, so just guessing. `m` is the convention used for the logger though. No idea why that is either, but one of them will have to change.

Reply:
Oops, bad variables from me...

It should be `scope`, `method`, and `args` - the extension host sends us requests of the form:
`[scope, method, argsAsJson]`, like: `["MainThreadDiagnostics", "$changeMany", diagnosticsJson]`

Reply:
Fixed in https://github.com/onivim/oni2/pull/999/commits/81033d07085beb2ebf9d234d927f42fc0f8ee0cd